src/loaders/OMETiffLoader.js

```diff
@@ -46,7 +46,9 @@ export default class OMETiffLoader {
     this.height = this.omexml.SizeY;
     this.tileSize = firstImage.getTileWidth();
     const { SubIFDs } = firstImage.fileDirectory;
-    this.numLevels = SubIFDs?.length || this.omexml.getNumberOfImages();
+    this.numLevels = SubIFDs?.length
+      ? SubIFDs.length + 1
+      : this.omexml.getNumberOfImages();
```

Comment:
This resolves a small bug I noticed and mentioned [here](https://github.com/hms-dbmi/viv/pull/305#issue-530552515)